src/main/scala/uk/gov/nationalarchives/tdr/api/http/Routes.scala

```diff
@@ -23,6 +25,29 @@ class Routes(val config: Config) extends Cors {
   val ttlSeconds: Int = 10
   val url: String = config.getString("auth.url")
 
+  def unmarshalEntity(entity: ResponseEntity): String = {
+    Unmarshal(entity).to[String].toString
+  }
+
+  def isNon200Status(status: StatusCode): Boolean = {
+    val code = status.intValue.toString
+    !code.startsWith("2")
+  }
+
+  def logging: Directive0 = {
+    def logNon200Response(req: HttpRequest)(res: RouteResult): Unit = {
+      res match {
+        case Complete(resp) =>
+          if (isNon200Status(resp.status)) {
+            val responseEntity = unmarshalEntity(resp.entity)
+            logger.info(s"Non 200 Response: $req\nStatus Code: ${resp.status}\nResponse Entity: $responseEntity")
+          }
+        case Rejected(reason) =>
+          logger.info(s"Rejected Reason: " + reason.mkString(", "))
```

Comment:
Have you got any thoughts about the level we use for these messages? Given that we control the clients, maybe this should be at least a warning? Genuinely not sure.

Reply:
Feels like Warning makes the most sense to me, so changed to warning.